charts/datadog/values.yaml

```
@@ -782,6 +782,9 @@ datadog:
       # datadog.securityAgent.runtime.fimEnabled -- Set to true to enable Cloud Workload Security (CWS) File Integrity Monitoring
       fimEnabled: false
 
+      # datadog.securityAgent.runtime.useSecruntimeTrack -- Set to true to send Cloud Workload Security (CWS) events directly to the new improved Agent events explorer
```

```suggestion
      # datadog.securityAgent.runtime.useSecruntimeTrack -- Set to true to send Cloud Workload Security (CWS) events directly to the Agent events explorer
```
Noticed another issue. The Datadog style guide says that our public documentation must be timeless and free of marketing speak, so we stay away from saying that things are new or improved.

Thank you for contributing to the public documentation!

---

applied in the last push